src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -131,7 +131,25 @@ def _bigquery(self) -> bigquery.Client:
             return bigquery.Client(project=self._source.project)
 
     def _run_sql(self, query: str) -> BigQueryRows:
-        return self._bigquery.query(query)
+        delays = (10, 20, 40, 80, None)
+        max_attempts = len(delays)
+        assert sum(delays[:-1]) < config.contribution_lambda_timeout
+        for attempt, delay in enumerate(delays, start=1):
```

Comment:
A 1-based loop variable is cardinal sin.

Reply:
Almost lost to temptation there. 👀